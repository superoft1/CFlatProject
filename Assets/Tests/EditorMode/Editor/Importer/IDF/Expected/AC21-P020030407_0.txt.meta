fileFormatVersion: 2
guid: 2f757decd84baa2418cb3adb4be22663
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
