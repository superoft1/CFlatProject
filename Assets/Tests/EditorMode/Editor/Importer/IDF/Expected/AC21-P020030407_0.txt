WeldNeckFlange, P61B-003LX, (0.0, -1.0, 0.0), (-11943.9, 9266.9, 100.9), False
(-11943.9, 9266.8, 100.9)
(-11943.9, 9266.9, 100.9)
FreeVertex: 1
Elbow90, P02A-000M3, (0.0, 0.0, -1.0), (-11943.9, 9267.3, 100.9), False
(-11943.9, 9266.9, 100.9)
(-11943.9, 9267.3, 101.3)
FreeVertex: 0
Pipe, P01A-0016U, (0.0, 0.0, 1.0), (-11943.9, 9267.3, 101.4), False
(-11943.9, 9267.3, 101.3)
(-11943.9, 9267.3, 101.4)
FreeVertex: 0
WeldOlet, P01A-0016U, (1.0, 0.0, 0.0), (-11943.9, 9267.3, 101.4), False
(-11943.9, 9267.3, 101.4)
(-11943.9, 9267.3, 101.4)
(-11943.7, 9267.3, 101.4)
FreeVertex: 1
Pipe, P01A-0016U, (0.0, 0.0, 1.0), (-11943.9, 9267.3, 101.5), False
(-11943.9, 9267.3, 101.4)
(-11943.9, 9267.3, 101.5)
FreeVertex: 0
WeldOlet, P01A-0016U, (0.0, 1.0, 0.0), (-11943.9, 9267.3, 101.5), False
(-11943.9, 9267.3, 101.5)
(-11943.9, 9267.3, 101.5)
(-11943.9, 9267.5, 101.5)
FreeVertex: 0
Pipe, P01A-000YX, (0.0, 1.0, 0.0), (-11943.9, 9267.5, 101.5), False
(-11943.9, 9267.5, 101.5)
(-11943.9, 9267.6, 101.5)
FreeVertex: 0
Pipe, P01A-000YX, (0.0, 1.0, 0.0), (-11943.9, 9267.6, 101.5), False
(-11943.9, 9267.6, 101.5)
(-11943.9, 9267.6, 101.5)
FreeVertex: 0
Elbow90, P02A-000CP, (0.0, 0.0, -1.0), (-11943.9, 9267.7, 101.5), False
(-11943.9, 9267.6, 101.5)
(-11943.9, 9267.7, 101.6)
FreeVertex: 0
WeldNeckFlange, P61B-001HW, (0.0, 0.0, 1.0), (-11943.9, 9267.7, 101.6), False
(-11943.9, 9267.7, 101.7)
(-11943.9, 9267.7, 101.6)
FreeVertex: 0
GateValve, P81C-005ZL, (0.0, 0.0, 1.0), (-11943.9, 9267.7, 101.8), False
(-11943.9, 9267.7, 101.9)
(-11943.9, 9267.7, 101.7)
FreeVertex: 0
Instrument, 020-PG-4018-DR, (0.0, 0.0, 1.0), (-11943.9, 9267.7, 101.9), False
(-11943.9, 9267.7, 101.9)
(-11943.9, 9267.7, 101.9)
FreeVertex: 0
Cap, P29A-0035U, (0.7, -0.7, 0.0), (-11943.9, 9267.8, 101.9), False
(-11943.9, 9267.8, 101.9)
FreeVertex: 1
Cap, P29A-0035U, (-0.7, -0.7, 0.0), (-11943.9, 9267.6, 101.9), False
(-11943.9, 9267.6, 101.9)
FreeVertex: 1
PressureGauge, 020-PG-4018, (0.0, 0.0, 1.0), (-11943.9, 9267.7, 102.0), False
(-11943.9, 9267.7, 102.0)
(-11943.9, 9267.7, 101.9)
FreeVertex: 1
Pipe, P01A-0016U, (0.0, 0.0, 1.0), (-11943.9, 9267.3, 101.9), False
(-11943.9, 9267.3, 101.5)
(-11943.9, 9267.3, 102.3)
FreeVertex: 1
FreeVertexTotal: 6
